Sources/SwiftKafka/Configuration/KafkaConfiguration.swift

```diff
@@ -14,159 +14,231 @@
 
 /// Collection of types used in the configuration structs this library provides.
 public enum KafkaConfiguration {
-    /// A Kafka Broker to connect to.
-    public struct Broker: Sendable, Hashable, CustomStringConvertible {
-        /// The host component of the broker to connect to.
-        public var host: String
+    /// The address of a Kafka broker.
+    public struct BrokerAddress: Sendable, Hashable, CustomStringConvertible {
+        /// The host component of the broker address.
+        public var host: String = "localhost"
 
         /// The port to connect to.
-        public var port: Int
+        public var port: Int = 9092
 
         public var description: String {
             "\(self.host):\(self.port)"
         }
 
-        public init(host: String, port: Int) {
-            self.host = host
-            self.port = port
-        }
+        public init() {}
```

Comment:
I think for this particular struct we should keep the init and not add any defaults since they don't make much sense or are these the defaults from rdkafka itself?